docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 * :bug:`8334` Importing addresses from MetaMask should work when multiple browser wallets are installed.
 * :bug:`-` rotki will now properly run background tasks when logging out and logging in again.
 * :bug:`-` rotki will now detect new tokens right after finishing decoding new events.
+* :bug:`-` The detection of accounts in new blockchains has been improved to avoid false positives when the account has been only spammed.
```

Comment:
```suggestion
* :bug:`-` The detection of account activity in new evm chains has been improved to avoid false positives when the account has been only sent spam tokens.
```